File: problems/algorithms/array-products/php/code/spec/products.spec.php
<?php
require __DIR__ . '/../src/products.php';

describe('Cart Tool', function () {

  it('counting total cart', function () {
    $products = [['Bicicleta', 1200.0], ['Capacete', 450.0]];
    expect(total($products))->toBe(1650.0);
  });
});

File: problems/algorithms/object-products-qtd/ruby/response/rspec/products.spec.rb
require_relative "../src/products"

produtos = [
  {
    "nome": 'Bicicleta',
    "preco": 1200.0,
  },
  {
    "nome": 'Capacete',
    "preco": 450.0,
  }
];

describe Product do
  describe ".total" do
    context "counting total cart " do
      it "return 1650.0" do
        expect(Product.total(produtos)).to eql(2)
      end
    end
  end
end
 
